locales/en.json

```diff
@@ -139,14 +139,16 @@
       "go-to-contributors": "Go to contributors",
       "more-info": "More information",
       "show-all": "Show all",
-      "remove-from-playlist": "Remove from playlist"
+      "remove-from-playlist": "Remove from playlist",
+      "transcription": "Transcription"
```

Comment:
can we use "Lyrics" for SongTreasures songs?
(Please double check what it's called in the app)